spec/sanity/events.js

```diff
@@ -328,7 +330,9 @@ describe('logs & events page', function() {
             //A search with 6 results
             events.filters.messageText.type('stopped');
             waitingForDebounce();
-            expect(events.mainTable.countRows()).toBe(6);
```

Comment:
Also you are right for removing it , yet this test don't check the data. we should check the message to see if it has the searched value.